.github/workflows/example.yml

```diff
@@ -237,6 +241,23 @@ jobs:
           path: ./Edk2
           key: edk2-${{ matrix.edk2-version }}
 
+      - name: Get versions of edk2
+        id: edk2_versions
+        run: |
+          echo "ver_current=$( echo ${{ matrix.edk2-version }} | tr -cd '0-9' )" >> "${GITHUB_OUTPUT}"
+          echo "ver_breaking=$( echo 'edk2-stable202305' | tr -cd '0-9' )" >> "${GITHUB_OUTPUT}"
+
+      - name: Use GCC5 for old edk2
+        id: gcc_toolchain
+        # GCC5 is depreciated since edk2-stable202305
```

Comment:
Just a nit: I never saw 'depreciated' in a programming context so I wondered if it means the same as 'deprecated'. According to this it does not: https://stackoverflow.com/a/9208164 

Given that I'm not a native English speaker I have no real preference here but it just seemed odd.

Reply:
Oh yeah :D
I did not relalise that those are 2 different terms xD
I assumed it is like `color` vs `colour`.